ee/models/assistant.py

```diff
@@ -11,8 +11,18 @@
 
 
 class Conversation(UUIDModel):
+    class Status(models.TextChoices):
+        NOT_STARTED = "not_started", "Not Started"
```

Comment:
If it exists, it WAS _started_, though it may not be _in-progress_. Maybe the right term here is "idle" or "dormant"?